exporter/datadogexporter/config_warnings_test.go

```diff
@@ -105,40 +105,16 @@ func TestPeerTags(t *testing.T) {
 		err                   string
 	}{
 		{
-			name: "both traces::peer_service_aggregation and traces::peer_tags_aggregation",
+			name: "traces::peer_tags_aggregation set to false",
```

Comment:
I think the whole test `TestPeerTags` can be removed. This is in `config_warnings_test.go` and is meant to test config warnings. Now there are no longer warnings